test/integration/multi-hmr.test.js

```diff
@@ -7,12 +7,13 @@ const execa = require('execa');
 const { browser } = require('../helpers/puppeteer');
 
 test('multi compiler', browser, async (t, page, util) => {
+  t.timeout(240000);
```

Comment:
Isn't it implied by flag in scripts?

Reply:
This was trial and error. Needs more work to get running with updated deps. Something about the new version of webpack I think is causing issues.